frontend/cypress/integration/featureFiles/waypoint.feature

```diff
@@ -35,62 +35,57 @@ Feature: Kiali Waypoint related features
     And the user hovers in the "ambient" label and sees "L7" in the tooltip
 
   @waypoint
-  @pft
   Scenario: User sees ztunnel traffic
     Given user is at the "graphpf" page
-    When user graphs "bookinfo" namespaces in the patternfly graph
+    When user graphs "bookinfo" namespaces
     Then user sees the "bookinfo" namespace
     Then user opens traffic menu
     And user "enables" "ambientZtunnel" traffic option
-    Then 7 edges appear in the patternfly graph
+    Then 7 edges appear in the graph
 
-  @waypoint
-  @pft
-  Scenario: User sees waypoint traffic
-    Given user is at the "graphpf" page
-    When user graphs "bookinfo" namespaces in the patternfly graph
-    Then user sees the "bookinfo" namespace
-    Then user opens traffic menu
-    And user "enables" "ambientWaypoint" traffic option
-    Then 11 edges appear in the patternfly graph
```

Comment:
This can be updated to 8 - Instead of 11

Reply:
The problem is that the number is not fixed, it changes from 8 to 10.

Examples:

https://github.com/kiali/kiali/actions/runs/11401957837/job/31726167454?pr=7799#step:8:679
https://github.com/kiali/kiali/actions/runs/11390256893/job/31711998051#step:8:680

Reply:
Right, this is why it checks that is at least this number, is not doing an exact match because of this